aci/resource_aci_cloudsubnet.go

```diff
@@ -259,8 +270,16 @@ func resourceAciCloudSubnetCreate(ctx context.Context, d *schema.ResourceData, m
 		cloudSubnetAttr.SubnetGroup = subnetGroup.(string)
 	}
 
-	cloudSubnet, err := aciClient.CreateCloudSubnet(ip, CloudCIDRPoolDn, desc, cloudSubnetAttr, zoneDn)
+	var cloudRsSubnetToCtx string
+	if relationTocloudRsSubnetToCtx, ok := d.GetOk("relation_cloud_rs_subnet_to_ctx"); ok {
+		cloudRsSubnetToCtx = GetMOName(relationTocloudRsSubnetToCtx.(string))
+	} else {
+		cloudRsSubnetToCtx = ""
+	}
```

Comment:
do we need this else statement? is the var initialisation not already setting it to ""

Reply:
True. I removed it